messenger.rst

```diff
@@ -529,84 +823,308 @@ you can disable them like this:
         // config/packages/messenger.php
         $container->loadFromExtension('framework', [
             'messenger' => [
-                'buses' => [
-                    'messenger.bus.default' => [
-                        'default_middleware' => false,
+                'transports' => [
+                    'async_priority_high' => 'dsn' => '%env(MESSENGER_TRANSPORT_DSN)%?queue_name=high_priority',
+                    'async_priority_low' => [
+                        'dsn' => '%env(MESSENGER_TRANSPORT_DSN)%',
+                        'options' => [
+                            'queue_name' => 'normal_priority'
+                        ]
                     ],
                 ],
             ],
         ]);
 
-Adding your own Middleware
-~~~~~~~~~~~~~~~~~~~~~~~~~~
+Options defined under ``options`` take precedence over ones defined in the DSN.
+
+==================  =================================== =======
+     Option               Description                   Default
+==================  =================================== =======
+table_name          Name of the table                   messenger_messages
+queue_name          Name of the queue (a column in the  default
+                    table, to use-use one table for
+                    multiple transports)
+redeliver_timeout   Timeout before retrying a messages  3600
```

Comment:
it's missing the unit. I guess it's seconds. I guess this should also be clarified in the code documenting the options